test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/SmokeTests.cs

```diff
@@ -98,4 +99,63 @@ public void SubmitLogs()
         collector.AssertExpectations();
     }
 #endif
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void TracesResource()
+    {
+        using var collector = new MockSpansCollector(Output);
+        SetExporter(collector);
+        collector.ResourceExpector.Expect("service.name", ServiceName);
+        collector.ResourceExpector.Expect("telemetry.sdk.name", "opentelemetry");
+        collector.ResourceExpector.Expect("telemetry.sdk.language", "dotnet");
+        collector.ResourceExpector.Expect("telemetry.sdk.version", typeof(global::OpenTelemetry.Resources.Resource).Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>()?.Version);
+        collector.ResourceExpector.Expect("telemetry.auto.version", "0.5.1-beta.1");
+        collector.ResourceExpector.Expect("splunk.distro.version", typeof(Plugin).Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>()?.Version);
+
+        SetEnvironmentVariable("OTEL_DOTNET_AUTO_TRACES_ADDITIONAL_SOURCES", "MyCompany.MyProduct.MyLibrary");
+        RunTestApplication();
+
+        collector.ResourceExpector.AssertExpectations();
+    }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void MetricsResource()
+    {
+        using var collector = new MockMetricsCollector(Output);
+        SetExporter(collector);
+        collector.ResourceExpector.Expect("service.name", ServiceName);
+        collector.ResourceExpector.Expect("telemetry.sdk.name", "opentelemetry");
+        collector.ResourceExpector.Expect("telemetry.sdk.language", "dotnet");
+        collector.ResourceExpector.Expect("telemetry.sdk.version", typeof(global::OpenTelemetry.Resources.Resource).Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>()?.Version);
+        collector.ResourceExpector.Expect("telemetry.auto.version", "0.5.1-beta.1");
```

Comment:
I do not found any good solution to take the version from the dependencies (there is no direct reference to AutoInstrumentation). I think it is good enough solution. We will have to change these value when upgrading OTel Auto.

Reply:
I think this will start failing just after publishing a new tag... Right? We will require to make 2 PRs during release. One before release and second after the release. I am not sure if we should use `MinVer` 😉

Reply:
It should be one PR, when you upgading OTelAutoInstrumentation.

Splunk distro version should be handled automatically.